src/server/debug/server/server.go

```diff
@@ -292,7 +292,16 @@ func (s *debugServer) makeTasks(ctx context.Context, request *debug.DumpV2Reques
 			}
 		}
 		if len(sys.Profiles) > 0 {
-			ts = append(ts, s.makeProfilesTask(server, sys.Profiles))
+			podCount := 0
+			for _, app := range sys.Profiles {
+				podCount += len(app.Pods)
```

Comment:
I suppose we could get more granularity than this; pods * # default profiles (2) * containers